src/core/DataFile.cpp

```diff
@@ -467,7 +467,17 @@ bool DataFile::copyResources(const QString& resourcesDir)
 				{
 					// Get absolute path to resource
 					bool error;
-					QString resPath = PathUtil::toAbsolute(el.attribute(*res), &error);
+					auto attribute = el.attribute(*res);
+
+					// Skip empty resources. In some cases, there might be an
+					// alternative way of actually representing the data (e.g. data element for SampleTCO)
+					if (attribute.isEmpty())
```

Comment:
Can you please explain exactly why this was necessary to be in this PR?

Reply:
Ah, this is mentioned in #5990 : https://github.com/lmms/lmms/pull/5990#issuecomment-841828782 . I still wonder if it could break anything.